content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,258 @@
 
-* A service for each CouchDB instance.
-* A CHT API service.
-* HAProxy Healthcheck service.
-* HAProxy service.
-* Upgrade Service service.
-* A CHT Sentinel service.
-* An nginx service to act as a reverse proxy and terminate TLS connections
+### MoH Kenya
+
+Kenya hosts 47 production instances in a Kenyan data center running [K3s](https://k3s.io/). They run a simplified k3s set up with 47 isolated installations of k3s to easy management complexity.  Thus each CHT instance is bound to the VM's local storage.  If a VM fails, there is no automatic fail-over and a restore from backup is needed.
+
+|                   |                             |
+|------------------:|:----------------------------|
+|    CHT Instances: | 47                          |
+|          Hosting: | k3s                         |
+|            Cores: | tk                          |
+|              RAM: | tk                          |
+|             Disk: | 4,700 GB  (Max 700, Min 21) |
+| Multi-Node Couch: | No                          |
+|            Users: | 123.3k                      |
+
+
+<!-- 
+sources:
+
+Jul 15th, 2023 Slack
+https://medic.slack.com/archives/CBQH2HNJC/p1689443385526589?thread_ts=1689373994.245809&cid=CBQH2HNJC
+
+May 12 2025 Google doc listing instances, users and disk use
+MoH Kenya Kubernetes Migration Schedule
+https://docs.google.com/spreadsheets/d/1m0TERssHNlJZ-tLdeDUkEKPP_9wr3_uPMlgcjoVbRjc/edit?gid=0#gid=0
+
+-->
+
+### MoH Uganda
+
+Uganda hosts 1 of production instance in a Uganda data center running [K3s](https://k3s.io/).
+
+|                   |                           |
+|------------------:|:--------------------------|
+|    CHT Instances: | 1                         |
+|          Hosting: | K3s on VMware)            |
+|            Cores: | 32 (4 VMs x 8 cores/ea)   |
+|              RAM: | 64 GB (4 VMs x 16GB/ea)   |
+|             Disk: | 600 GB (4 VMs x 150GB/ea) |
+| Multi-Node Couch: | Yes                       |
+|            Users: | 12k                       |
+
+<!-- 
+sources:
+
+May 2nd,2024 Slack
+https://medic.slack.com/archives/C06TP97HRMZ/p1714639214379159
+
+-->
+
+
+### Medic
+
+Medic hosts 46 production instances on Amazon [Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) (EKS).
+
+|                   |                                                 |
+|------------------:|:------------------------------------------------|
+|    CHT Instances: | 46                                              |
+|          Hosting: | k8s on EKS                                      |
+|            Cores: | 64 (8 VMs x 8 Cores/ea)                         |
+|              RAM: | 256 GB (8 VMs x 32GB/ea)                        |
+|             Disk: | 8.6 TB (8 VMS x 500GB/ea + 46 CHTs x ~100GB/ea) |
+| Multi-Node Couch: | 44 No, 2 Yes                                    |
+|            Users: | 25.7k                                           |
+
+
+<!-- 
```

Comment:
@mrjones-plip, if I'm not mistaken, this is not public data, so we shouldn't add it to our public documentation repository. Please remove it from this file until we receive explicit consent from MoHes to publish information about their deployments.